### ISSUE TITLE
MN reward diffrence

### ISSUE THREAD

**Issue by skironDotNet** [syscoin/syscoin#302]
I have old MN and new MN in terms of when started and new gives more than old, I remember reading about increased rewards but my old MN never got any. Do I have to restart the MN or sent funds to new address an then start?
![sys mn](https://user-images.githubusercontent.com/2981715/49172849-b1510e00-f307-11e8-9a6a-0995814a10a6.png)

**Comment by sidhujag** [comment]
That is your seniority kicking in, your new masternode must be based on older 100k tx?

**Comment by skironDotNet** [comment]
I don't think you have it right, the old node gives less than new, how this is seniority?

Old means the node started in May 2018. New means started 2-4 weeks ago. So I think old node is more senior and should give more, and "new" could give 25, so it makes no sense. In fact I think something is wrong with the "old" node that it never started making more

**Comment by locutus75** [comment]
Need more information; you joined on Slack? or Telegram? You can send me a DM on both. (locutus)

**Comment by sidhujag** [comment]
need the tx collateral backing the MN to know

**Comment by skironDotNet** [comment]
I started looking up the TX and addresses and...
I'm so sorry, my bad, very stupid error on my side. So I had original MN that I broke when SYS spiked on Binance, but never sold, so created back the mn under new address, and called is mn1-new. the old address I called mn1-disabled. Then when created 2nd MN I didn't really named it and I thought new MN is "old" and the label for "old" mn mislead me as it was new MN, this it looked to me like new makes more than old.

Seniority it is.
